packages/mdc-textfield/index.js

```diff
@@ -103,6 +156,15 @@ export class MDCTextfield extends MDCComponent {
     };
   }
 
+  /**
+   * @return {!{
```

Comment:
Should we create `@record` definitions for these collections of methods?

Reply:
These subsets of the adapter API are kind of arbitrary (I assume we only split it like this as an attempt at readability); the only thing that really matters is the collective adapter API. Kind of unfortunate that we have to annotate these at all...